src/main/frontend/extensions/zotero/extractor.cljs

```diff
@@ -91,6 +92,10 @@
      (util/format "![%s](%s)" label link)
      (util/format "[%s](%s)" label link))))
 
+(defn org-link
```

Comment:
This function is not multi-arity, we can remove the `()` pair enclosing the args and fn body.
(defn org-link
  [label link]
   (util/format "[[%s][%s]]" link label))

Reply:
You are totally right. Thanks, will revise them.